musl doesn't have posix_spawn_file_actions_addclosefrom_np, so python doesn't
have os.POSIX_SPAWN_CLOSEFROM on musl.

see: https://github.com/qtile/qtile/issues/5210

--- a/libqtile/core/manager.py
+++ b/libqtile/core/manager.py
@@ -1331,7 +1331,7 @@
                 (os.POSIX_SPAWN_DUP2, 2, null.fileno()),
             ]
 
-            if sys.version_info.major >= 3 and sys.version_info.minor >= 13:
+            if hasattr(os, "POSIX_SPAWN_CLOSEFROM"):
                 # we should close all fds so that child processes don't
                 # accidentally write to our x11 event loop or whatever; we never
                 # used to do this, so it seems fine to only do it on python 3.13 or
